src/main/scala/de/frosner/ddq/core/Check.scala

```diff
@@ -264,7 +265,7 @@ object Check {
       if (succeedingRows == count)
         ConstraintSuccess(s"Constraint $constraintString is satisfied")
       else
-        ConstraintFailure(s"${count - succeedingRows} rows did not satisfy constraint $constraintString")
+        ConstraintFailure(s"${if((count - succeedingRows) == 1) "One row" else s"${count - succeedingRows} rows"} did not satisfy constraint $constraintString")
```

Comment:
@bkomboz you can factor out `count - succeedingRows` as `failingRows` for example to avoid code duplication.